bubblewrap.c

```diff
@@ -1869,7 +1869,7 @@ main (int    argc,
   acquire_privs ();
 
   /* Never gain any more privs during exec */
-  if (prctl (PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0) < 0)
+  if (is_privileged && prctl (PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0) < 0)
```

Comment:
NO_NEW_PRIVS affects other things than setuid. For instance, a file could be setcaps to give you capabilities even if you don't get uid 0, and it affects how LSMs handle exec. It also enables the use of seccomp filters, and in the future could trigger other security aspects.

We can move the setting of this to a later time, which *probabl* fixes the problem, but I'm still pretty scared that it could be missed somehow, and would prefer if we could call the setuid helpers from outside the sandbox.